### PR TITLE
Update the UI to be able to delete a newly created static section type

### PR DESCRIPTION
**Pull request by mahfouzhseineweev**

### CODE REVIEW

File: src/base/index.vue [hunk 1]
```diff
@@ -1031,6 +1039,31 @@ export default {
...
+      axios
+          .delete(URL, config)
+          .then((res) => {
+            this.showToast(
+                "Success",
+                "info",
+                res.data.message
+            );
+            this.types.splice(index, 1)
+            this.$emit("load", true);
```
[the remark this answers is not in the document]
Comment (mahfouzhseineweev) [reply src/base/index.vue:1060]: Updated

File: src/base/index.vue [hunk 2]
```diff
@@ -1031,6 +1039,31 @@ export default {
...
+    deleteSectionType(sectionTypeName, index) {
+      const token = this.$cookies.get("sections-auth-token");
+      const config = {
+        headers: sectionHeader(({origin: this.$sections.projectUrl, token})),
+      };
+      const URL =
+          this.$sections.serverUrl +
+          `/project/${this.$sections.projectId}/section-types/${sectionTypeName}`;
+
+      axios
```
[the remark this answers is not in the document]
Comment (mahfouzhseineweev) [reply src/base/index.vue:1051]: Updated

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
-### 1.0.8
+### 1.0.15
```
[the remark this answers is not in the document]
Comment (mahfouzhseineweev) [reply CHANGELOG.md]: Updated

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@geeks-solutions/vue-sections",
-  "version": "1.0.8",
+  "version": "1.0.15",
```
[the remark this answers is not in the document]
Comment (mahfouzhseineweev) [reply package.json]: Updated

File: src/base/index.vue [hunk 5]
```diff
@@ -131,16 +131,24 @@
...
               :key="type.name"
-              @click="currentSection = type"
             >
-              <SectionItem
-                v-if="type.name && !type.name.includes('local')"
-                class="bg-light-blue "
-                :title="formatName(type.name)"
-                :icon="type.name"
-              />
+              <div class="section-delete">
```
[the remark this answers is not in the document]
Comment (jfayad) [reply src/base/index.vue:137]: The confirmation is good.

The icon, I think it's better if we re-use the same icon being used in the edit page mode (to delete a section)

[the remark this answers is not in the document]
Comment (mahfouzhseineweev) [reply src/base/index.vue:137]: Updated

Screen Shot 2022-06-01 at 12 27 38 PM

[the remark this answers is not in the document]
Comment (mahfouzhseineweev) [reply src/base/index.vue]: Updated the UI to Hide delete icon for non private static types @jfayad

**Pull request merged**